civihr_default_theme/scss/mail.scss

```diff
@@ -55,19 +55,28 @@ table.social {
 
 }
 .social .soc-btn {
-  padding: 3px 7px;
+  padding: 5px 7px 0px 7px;
   font-size:12px;
-  margin-bottom:10px;
   text-decoration:none;
-  color: #FFF;font-weight:bold;
-  display:block;
+  color: #FFF;
+  font-weight:bold;
   text-align:center;
+  display:table-cell;
+  width:100%;
+  min-width:232px;
 }
+
 a.fb { background-color: #3B5998!important; }
 a.tw { background-color: #1daced!important; }
 a.gp { background-color: #DB4A39!important; }
 a.ms { background-color: #000!important; }
 
+.soc-td{
+  background-color: #DB4A39;
+  padding:0 !important;
```

Comment:
For Mac Thunderbird we have used important, In manager detail section, Email id is going out side the gray box if we are not using important.